Makefile

```diff
@@ -7,7 +7,7 @@ MAIN        = $(SRC_DIR)/main.cpp
 SRCS        = $(shell find $(SRC_DIR) -name "*.cpp" -type f)
 LIB_SRCS    = $(filter-out $(MAIN), $(SRCS))
 TESTS       = $(wildcard tests/*.cpp)
-EXAMPLES    = $(wildcard examples/*.cpp)
+EXAMPLES    = $(shell find examples -name "*.cpp" -type f)
```

Comment:
That fixes the problem I mentioned earlier, @wesleycsj

Reply:
Beautiful.

Reply:
Oh, you are alive @rafaelcn 

@wesleycsj was asking for you another day.